cvat-ui/src/components/file-manager/external-image-selector.tsx

```diff
@@ -117,28 +131,43 @@ function processRecordingData(data) {
     return data.record.cam.map(item => {
         const { width, height } = item.params;
         const frames = item.abs_timestamp_ns.map(padFrameName);
-        const indexByFrame = {};
-        item.abs_timestamp_ns.forEach((f, i) => indexByFrame[f] = i);
-        return { width, height, frames, indexByFrame };
-    })
+        const sourceFrames = item.abs_timestamp_ns.map(e => +e);
```

Comment:
be careful: absolute timestamps are too large to be represented exactly as JS number, that's why they are passed as strings. Unless you need to do arithmetic on them I suggest leaving them as strings, you can sort alphanumerically - they are all the same length. If you need to do arithmetic use BigInt.

Reply:
Removed the string to number conversion.